manifests/server/online-train/patch-trainer.yaml

```diff
@@ -19,16 +19,16 @@ spec:
   template:
     spec:
       containers:
-      - name: server-api
-      - name: online-trainer
-        image: kepler_model_server
-        imagePullPolicy: IfNotPresent
-        volumeMounts:
-          - name: cfm
-            mountPath: /etc/kepler/kepler.config
-            readOnly: true
-          - name: mnt
-            mountPath: /mnt
-            readOnly: false
-        command: [ "python3.10" ]
-        args: ["-u", "src/train/online_trainer.py" ]
+        - name: server-api
+        - name: online-trainer
+          image: kepler_model_server
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: cfm
+              mountPath: /etc/kepler/kepler.config
+              readOnly: true
+            - name: mnt
+              mountPath: /mnt
+              readOnly: false
+          command: [python3.10]
+          args: [-u, src/train/online_trainer.py]
```

Comment:
could you please verify if this change in indentation (scope) does not break anything?